onnx/backend/test/case/node/image_decoder.py

```diff
@@ -1,16 +1,19 @@
 # Copyright (c) ONNX Project Contributors
 #
 # SPDX-License-Identifier: Apache-2.0
+from __future__ import annotations
+
+import io
 
-import cv2
 import numpy as np
+import PIL.Image
 
 import onnx
 from onnx.backend.test.case.base import Base
 from onnx.backend.test.case.node import expect
 
 
-def generate_checkerboard(width, height, square_size):
+def generate_checkerboard(width: int, height: int, square_size: int):
```

Comment:
What about annotating the return as well?

Reply:
Done